src/utils/libfabric/libfabric_rail.cpp

```
@@ -1267,6 +1267,15 @@ nixlLibfabricRail::postRead(void *local_buffer,
 
 // Memory Registration Methods
 
+// Memory Registration Resource Cache (MRRC) implementation
+// Based on He et al. "An efficient design for fast memory registration in RDMA" (JNCA 2009)
+
+size_t
+nixlLibfabricRail::getMRCacheSize() const {
```

Cppcheck (2.20.0)

[style] The function 'getMRCacheSize' is never used.

(unusedFunction)